lib/modules/dosomething/dosomething_global/dosomething_global.module

```diff
@@ -269,6 +298,24 @@ function dosomething_global_convert_country_to_language($country) {
 }
 
 /**
+ * Converts the given language into its assigned URL prefix.
+ *
+ * @param string language
+ *  The language to get the prefix for (eg: 'en', 'mx')
+ *
+ * @return
+ *  The URL prefix mapped to this language
+ */
+function dosomething_global_get_prefix_for_language($language) {
```

Comment:
oh thats better, for some reason i thought language list only returned a list of the languages, not all of the details. Ill switch this out!